src/templates/static/_layout/layout.jsx

```diff
@@ -54,7 +54,7 @@ const Layout = () => {
                         </div>
                         <Footer />
                     </div>
-                    <InterviewPopup /> {/* TODO: remove when UX research is finished */}
+                    <InterviewPopup />
```

Comment:
let's keep the comments for these two, we still want to remove them when the UX research is finished.